containers/Dockerfile.buildenv

```diff
@@ -65,11 +65,14 @@ RUN apt-get update -y \
 RUN apt-get install -y ccache doxygen
 
 # Install Python packages
+# We only install packages that are needed by the build system (e.g. to compile
+# Python bindings or build documentation) or used by Python code that is
+# unit-tested. Any other packages can be installed on-demand.
```

Comment:
Could you specify that `We need coverxygen, beatifulsoup4, and pybtex for both python2 and python3 so that people can build the documentation with either python 2 or 3.`